DigitalLearningSolutions.Web/ViewModels/Frameworks/FrameworksViewModel.cs

```diff
@@ -1,18 +1,23 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.Frameworks
 {
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+
     public class FrameworksViewModel
     {
-        public FrameworksViewModel(bool isFrameworkDeveloper, bool isFrameworkContributor, MyFrameworksViewModel myFrameworksViewModel, AllFrameworksViewModel allFrameworksViewModel)
+        public FrameworksViewModel(bool isFrameworkDeveloper, bool isFrameworkContributor, MyFrameworksViewModel myFrameworksViewModel, AllFrameworksViewModel allFrameworksViewModel, FrameworksTab currentTab)
```

Comment:
Might want to reformat this

Reply:
Reformatted along with cleaning up the line endings here and the other file.